### PR TITLE
Updated Readme file

### PR DESCRIPTION
**Pull request by BuggyMess**
added some useless comments to the Readme file

### CODE REVIEW

**Comment by SandwichProtectorIsDead** [comment]
why so serious ?

**Review by SandwichProtectorIsDead — COMMENTED** [review]
nice empire build you got there

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /*
 this is my first contribution to this project 1
+*************another master update******************
```
Comment (SandwichProtectorIsDead) [inline README.md:3]: wtf is this right here ?

Comment (SandwichProtectorIsDead) [reply README.md:3]: a title

File: README.md [hunk 2]
```diff
@@ -1 +1,18 @@
+/*
+this is my first contribution to this project 1
+*************another master update******************
```
Comment (SandwichProtectorIsDead) [inline README.md:3]: wtf is all of this

File: README.md (hunk 1 above)
Comment (SandwichProtectorIsDead) [reply README.md:3]: whattt

**Comment by BuggyMess** [comment]
this is a comment

**Comment by BuggyMess** [comment]
wahhhht

**Comment by BuggyMess** [comment]
ql²efjqljfqlmjf

**Comment by BuggyMess** [comment]
lkqdsjflqsjf

File: README.md (hunk 1 above)
Comment (BuggyMess) [reply README.md:3]: this is **brilliant__**

File: README.md (hunk 2 above)
Comment (BuggyMess) [reply README.md:3]: huh ?

**Pull request merged**